proofpoint-threat-response/info.json

```diff
@@ -0,0 +1,883 @@
+        }
+      ],
+      "enabled": true
+    },
+    {
+      "operation": "block_hash",
+      "title": "Block Hash",
+      "category": "containment",
+      "annotation": "block_hash",
+      "description": "Block the supplied file hashes to the specified file hash block list.",
+      "output_schema": [],
+      "parameters": [
+        {
+          "name": "indicator",
+          "title": "Hash",
+          "type": "text",
+          "description": "Specify a comma-separated list of file hashes to add to the file hash block list.",
+          "tooltip": "Specify a comma-separated list of file hashes to add to the file hash block list.",
+          "required": true,
+          "editable": true,
+          "visible": true
+        },
+        {
+          "name": "list_id",
+          "title": "Blacklist ID",
+          "type": "text",
+          "description": "Specify the Id of hashes block list.",
+          "tooltip": "Specify the Id of hashes block list.",
+          "required": true,
+          "editable": true,
+          "visible": true
+        },
+        {
+          "name": "expiration",
+          "title": "Expiration",
+          "type": "datetime",
+          "description": "Specify the date and time the supplied file hashes should be removed from the block list.",
+          "tooltip": "Specify the date and time the supplied file hashes should be removed from the block list.",
+          "required": false,
+          "editable": true,
+          "visible": true
+        }
+      ],
+      "enabled": true
+    },
+    {
+      "operation": "search_indicator",
+      "title": "Search Indicator",
+      "category": "investigation",
+      "annotation": "search_indicator",
+      "description": "Get the indicators from the specified list, according to the defined filter.",
+      "output_schema": [],
+      "parameters": [
+        {
+          "name": "filter",
+          "title": "Filter",
+          "type": "text",
+          "description": "Specify the filter for the indicator search. e.g. For example, \"1.1\" will return [1.1.1.1, 22.22.1.1, 1.1.22.22]",
+          "tooltip": "Specify the filter for the indicator search. e.g. For example, \"1.1\" will return [1.1.1.1, 22.22.1.1, 1.1.22.22]",
+          "required": false,
+          "editable": true,
+          "visible": true
+        },
+        {
+          "name": "list_id",
+          "title": "Blacklist ID",
+          "type": "text",
+          "description": "Specify the ID of the list in which to search.",
+          "tooltip": "Specify the ID of the list in which to search.",
+          "required": true,
+          "editable": true,
+          "visible": true
+        }
+      ],
+      "enabled": true
+    },
+    {
+      "operation": "delete_indicator",
+      "title": "Delete Indicator",
+      "http_method": "DELETE",
+      "category": "investigation",
+      "annotation": "delete_indicator",
+      "description": "Delete Indicator",
```

Comment:
Please update the description: Removes an indicator from Proofpoint Threat Response based on the input parameters.